packages/react-badge/package.json

```diff
@@ -43,6 +43,7 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
+    "@fluentui/react-icons": "^1.1.140",
```

Comment:
I would recommend using version `2.0.151-beta.2`. This has the resizable icons and fixes some minor visual regressions

Reply:
So either you would need to get your PR to bump to v2-beta for react-icons first or add the changes from this PR to yours

either works